modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -314,13 +317,23 @@ function FieldBase({
 							>
 								{label && showLabel && label}
```

Comment:
Use `{showLabel && label}` instead of `{label && showLabel && label}`

Reply:
Done. See it here https://github.com/liferay-forms/liferay-portal/pull/484/commits/4a319cce54f917cfecbd62c1bda54262e93cc6b2